pkg/adaptation/result.go

```diff
@@ -90,7 +90,8 @@ func collectCreateContainerResult(request *CreateContainerRequest) *result {
 				Hooks:       &Hooks{},
 				Rlimits:     []*POSIXRlimit{},
 				Linux: &LinuxContainerAdjustment{
-					Devices: []*LinuxDevice{},
+					Devices:    []*LinuxDevice{},
+					CDIDevices: []*CDIDevice{},
```

Comment:
`CDIDevices` should be at the same level as `Mounts` and envs. They are not Linux-specific.

Reply:
Updated accordingly.